src/bin/pg_dump/pg_backup_archiver.c

```diff
@@ -3038,8 +3038,14 @@ _tocEntryRequired(TocEntry *te, teSection curSection, ArchiveHandle *AH)
 	if ((strcmp(te->desc, "<Init>") == 0) && (strcmp(te->tag, "Max OID") == 0))
 		return 0;
 
-	/* Mask it if we only want schema */
-	if (ropt->schemaOnly)
+	/*
+	 * Mask it if we only want schema.
```

Comment:
this is confusing, we want beahvior change for config tables but wouldn't it apply to all the tables?